keyboards/handwired/colorlice/rules.mk

```diff
@@ -0,0 +1,36 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
+LTO_ENABLE = yes            # Use link time optimization
+RGB_MATRIX_ENABLE = WS2812	# Enable keyboard RGB backlight functionality with ws2812 driver
+RGBLIGHT_ENABLE = no		# Enable keyboard RGB underglow
+NO_USB_STARTUP_CHECK = no	# Disable initialization only when usb is plugged in
```

Comment:
Could you please change Lines 34-36 to space indent instead of tabs?

```Makefile
RGB_MATRIX_ENABLE = WS2812  # Enable keyboard RGB backlight functionality with ws2812 driver
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
NO_USB_STARTUP_CHECK = no   # Disable initialization only when usb is plugged in
```